example/src/main/assets/licenses.html

```diff
@@ -43,14 +43,44 @@
     list-style-type: lower-alpha;
     padding-left: 2em;
     }
+    input ~ .license {
+    max-height: 200px;
+    overflow: hidden;
+    }
+    input, label {
+    position: absolute;
+    right: 16px;
+    width: 12px;
+    height: 12px;
+    margin: 20px;
+    }
+    input {
+    opacity: 0;
+    z-index: 1;
+    }
+    label {
+    margin-top: 16px;
+    text-align: center;
+    font-weight: bold;
+    }
+    label::before {
+    content: "+";
+    }
+    input:checked + label::before {
+    content: "-";
+    }
+    input:checked ~ .license {
+    max-height: none;
+    }
   </style>
 </head>
 <body>
     <div class="library">
       <!-- https://opensource.org/licenses/Apache-2.0 -->
-      <h1 class="title">Android Orma</h1>
-      <p class="notice">Copyright (c) 2015 FUJI Goro (gfx)<br/>SQLite.g4 is: Copyright (c) 2014 by Bart Kiers<br/></p>
-      <p><a href="https://github.com/ReactiveX/RxJava">https://github.com/ReactiveX/RxJava</a></p>
+      <h1 class="title">Android Support Libraries</h1>
```

Comment:
I don't know why this change is included.
I just executed `./gradlew generateLicensePage` to generate `license.html`.
`licenses.html` was not updated to the latest state?